data/hai/hai missions.txt

```diff
@@ -2099,3 +2099,213 @@ mission "Hai Pilgrimage"
+				`	"Let's go find her!"`
+					goto accept
+				`	"I have other places to be."`
+					decline
+			label tons
+			`	"Well, I... I don't know... I cook when I'm worried, and Anoo has been missing for three months! I've vacuum-sealed all three tons of cookies. They should last a year or two. Maybe she'll share them with her friends? She has many friends! Tons of friends! They all miss her, but none are willing to help...`
+				goto questions
+			label gone
+			`	"Anoo graduated from a medical academy on Frostmark four months ago, near the top of her class! A prominent hospital on the Hai-home needed a new trauma surgeon and said she would be perfect for the job. She accepted the position and said she would be on vacation for a few weeks.`
+			`	"A month later, she told them she won't be coming, and nobody has heard from her since.`
+				goto questions
+			label authorities
+			`	"I talked to the Central Medical Bureau, and they told me she got clearance for practicing medicine anywhere in Hai space or beyond. They've heard nothing from her since she got clearance. She has to register new prescriptions with the Bureau, so they should have heard from her.`
+			`	"The police told me that several other people have reported Anoo missing, but they refused to tell me anything else. They said I had to talk to military, but they won't tell me anything.`
+			`	"Maybe she's involved in something secret? Or illegal? No, she'd never do that. Where is she?`
+				goto questions
+			label friends
+			`	"I have few friends, and none of them are interested in finding a missing Hai who could be anywhere. None of my customers want to help either, and her family is all dead.`
+				goto questions
+			label accept
+			`	"She always talked about doing volunteer work for those who couldn't afford medical treatment. When she was little, her mother would take her to Stonebreak to help at local shelters. Let's start there."`
+				accept
+	on visit
+		dialog "You forgot Eeeya or some of her mountains of cookies. Maybe they're in one of your escorts?"
+
+mission "Acorn Delights: Part 2"
+	landing
+	name "Acorns for Anoo"
+	description "Bring Eeeya to <destination> to deliver <cargo> to her missing granddaughter."
+	source "Stonebreak"
+	destination "Cloudfire"
+	passengers 1
+	cargo "acorn cookies" 3
+	to offer
+		has "Acorn Delights: Part 1: done"
+	on offer
+		log "Took a Hai lady named Eeeya to Stonebreak to help find her granddaughter, who is named Anoo."
+		conversation
+			`You spend several hours flying Eeeya between the massive communities of Stonebreak. She checks every homeless shelter, volunteer ward, women's shelter, and low-income hospital. The story is always the same: Anoo hasn't been seen in years, though some remember the energetic little girl in her youth before her mother left.`
+			`	After a few moments of crushing despair, Eeeya's ears perk up, and she says, "Since it's obvious she's not here, let's check on <destination> - she could be in military."`
```

Comment:
```suggestion
			`	After a few moments of crushing despair, Eeeya's ears perk up, and she says, "Since it's clear she's not here, let's check on <destination> - she could be in military."`
```